hera_qm/xrfi.py

```diff
@@ -1648,7 +1648,7 @@ def xrfi_run_step(uv_files=None, uv=None, uvf_apriori=None,
             # if uvf_apriori is not None and we wish to apply it to uv, do so here.
             elif apply_uvf_apriori:
                 flag_apply(uvf_apriori, uv, keep_existing=True, run_check=run_check,
-                           check_extra=check_extra,
+                           check_extra=check_extra, force_pol=True,
```

Comment:
I'm wondering whether there's ever a reason why we wouldn't want to `force_pol=True`. I can't think of any, but if you can, maybe expose this to the larger function with `True` as the default?

Reply:
`force_pol=True` is invoked here because `uvf_apriori` is a waterfall and will cause errors if `uv` has more then one pol. This made it past the unittests for some reason (are the unittests only involving single pol files?)